frontend/src/app/pages/marker-page/marker-page.component.html

```diff
@@ -55,11 +57,11 @@ <h3>Actions</h3>
             </div>
         </div>
 
-        <h3>Your Labels</h3>
+        <h3 class="category__name">Labels</h3>
```

Comment:
Probably line 48 also needs to be changed 😉

Look:
<img width="302" alt="screenshot 2018-11-11 at 21 43 24" src="https://user-images.githubusercontent.com/6514553/48318042-051ae200-e5fb-11e8-8094-b977abceb8b8.png">

Reply:
BTW. Tools now differ only by their icon color (white vs. black). Previously, they could also be distingushed by background color (blue vs. orange). Isn't it a regression? Previously it looked better for me 😉

Reply:
BTW. This label also looks weird now. For me, its font should be lighter.

Your proposal:
<img width="314" alt="screenshot 2018-11-11 at 21 50 36" src="https://user-images.githubusercontent.com/6514553/48318133-20d2b800-e5fc-11e8-9c94-3ac756a19012.png">

My proposal:
<img width="313" alt="screenshot 2018-11-11 at 21 50 27" src="https://user-images.githubusercontent.com/6514553/48318129-1ca69a80-e5fc-11e8-9bfe-d950321c58ea.png">